lib/aruba/platforms/windows_command_string.rb

```diff
@@ -16,7 +16,8 @@ def initialize(cmd)
 
       # Convert to array
       def to_a
-        Shellwords.split __getobj__
+        Shellwords.split( __getobj__.gsub('\\', 'ARUBA_TMP_PATHSEPARATOR') ).
```

Comment:
Yes, shellwords swallows windows path separators:

irb(main):001:0> require 'shellwords'
=> true
irb(main):002:0> Shellwords.split('c:\\windows\\system32')
=> ["c:windowssystem32"]

The solution in this PR is not very elegant, but it looks safe in given context.

A few more interesting observations:

This does not work:

irb(main):003:0> Shellwords.split('c:\\windows\\system32'.gsub('\\', '\\\\'))
=> ["c:windowssystem32"]

While this does work:

irb(main):004:0> Shellwords.split('c:\\windows\\system32'.gsub('\\', '\\\\\\\\'))
=> ["c:\\windows\\system32"]

So it might be solution as well, however it looks too subtle...